org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/LanguageServerImpl.xtend

```diff
@@ -107,16 +109,16 @@ import static org.eclipse.xtext.diagnostics.Severity.*
 @Log class LanguageServerImpl implements LanguageServer, WorkspaceService, TextDocumentService, LanguageClientAware, Endpoint, JsonRpcMethodProvider, IBuildListener {
 
 	@Inject @Accessors(PUBLIC_GETTER) RequestManager requestManager
-	@Inject WorkspaceSymbolService workspaceSymbolService
-	@Inject extension UriExtensions
-	@Inject extension IResourceServiceProvider.Registry languagesRegistry
-	@Inject ExecutableCommandRegistry commandRegistry
-	@Inject SemanticHighlightingRegistry semanticHighlightingRegistry
-	@Inject ILanguageServerShutdownAndExitHandler shutdownAndExitHandler
+	@Inject @Accessors(PROTECTED_GETTER) WorkspaceSymbolService workspaceSymbolService
```

Comment:
do we need `@since 2.16` for these?

Reply:
Good question. Should I rather use hand written methods then?

Reply:
yes

Reply:
Better like that?